mempool/src/tests/mod.rs

```diff
@@ -8,9 +8,6 @@ mod core_mempool_test;
 #[cfg(test)]
 mod shared_mempool_test;
 
-/// Mocks used for testing
+pub mod fuzzing;
```

Comment:
Interesting, why are the fuzzing mocks not featured under fuzzing?

Reply:
No idea 😄 Sounds like it might be worth looking into for a future clean up!